File: library/src/main/java/su/levenetc/android/textsurface/animations/Sequential.java
package su.levenetc.android.textsurface.animations;

import su.levenetc.android.textsurface.contants.TYPE;
import su.levenetc.android.textsurface.interfaces.ISurfaceAnimation;

/**
 * Created by Eugene Levenetc.
 */
public class Sequential extends AnimationsSet {
	public Sequential(ISurfaceAnimation... animations) {
		super(TYPE.SEQUENTIAL, animations);
	}
}

File: library/src/main/java/su/levenetc/android/textsurface/animations/Sequential.kt
package su.levenetc.android.textsurface.animations

import su.levenetc.android.textsurface.contants.TYPE
import su.levenetc.android.textsurface.interfaces.ISurfaceAnimation

/**
 * Created by Eugene Levenetc.
 */
open class Sequential(vararg animations: ISurfaceAnimation) : AnimationsSet(TYPE.SEQUENTIAL, *animations)